packages/web-react/src/types/shared/rest.ts

```diff
@@ -0,0 +1,5 @@
+type restValueType = unknown;
+
+export interface RestProps {
```

Comment:
To the naming. As we are referring to passing the props to children elements, the concept itself is called "Transferring Props". We are using, in most places, `restProps` or `otherProps` variable. But it seems to me that better naming for this type will be `TransferringProps` or `TransferProps`.

What do you think about it?

Reply:
In my opinion, it would be better if at least the basic set of properties could be taken from `DetailedHTMLProps<HTMLAttributes<HTMLSpanElement>, HTMLSpanElement>` in the case of Tag and so on. 
This would cover the default properties from the element without the need to define and type them.

Use these universal `RestProps` or `TransferringProps` only in the last row. But for now i will rename it to the `TransferringProps`.